zebra/zserv.c

```diff
@@ -524,6 +552,9 @@ static void zserv_process_messages(struct event *thread)
 	/* Reschedule ourselves if necessary */
 	if (need_resched)
 		zserv_event(client, ZSERV_PROCESS_MESSAGES);
+
+	/* Signal a wake up to client IO thread */
```

Comment:
this comment is a little misleading, imo. what we're doing is ensuring that the read socket is being included in the select/poll/etc. we're not forcing the read pthread to wake up.

Reply:
Will change this